### DISCUSSION TITLE
Units (dp and px)

### DISCUSSION THREAD

**Discussion by CyberCyclone** [GeekyAnts/NativeBase#5690] [General]
I'm trying to understand how the units work. I have a Box that has a width defined as `80` (no units defined). When I do an inspection on my iPad of the component it says the width is 320. 

When I use `PixelRatio.get()` it comes back as 2. If the units NativeBase uses are `dp` (independent pixels), shouldn't it be rendered as 160 (80 * 2) in literal pixels? How does it calculate it to be 320, which is 4 times 80?

Thanks!

**Comment by EricSoeun** [comment]
Hi! I think units in Native Base are calculated on a base four.

```
width="4" // 16px
width= "10" // 40px
etc.
```

I found this information here in the spacing file 👍 : https://github.com/GeekyAnts/NativeBase/blob/master/src/theme/base/space.ts

**Comment by CyberCyclone** [comment]
Thanks @EricSoeun. That's perfect 👍
